lib/hammer_cli_foreman/registration_tokens.rb

```diff
@@ -0,0 +1,25 @@
+module HammerCLIForeman
+  class RegistrationTokens < HammerCLIForeman::Command
+    resource :registration_tokens
+    command_name 'registration-tokens'
+    desc _('Manage registration tokens')
+
+    class InvalidateMultipleCommand < HammerCLIForeman::DeleteCommand
+      action :invalidate_jwt_tokens
+      command_name 'invalidate-multiple'
+      success_message _('Successfully invalidated registration tokens for %{users}.')
```

Comment:
`failure_message` is missing for both commands.

Reply:
We don't need a failure message because it will not fail in any case. We are handling other cases like missing params or no record found etc. We don't have a failure message for API as well. It can fail in UI but not in API.